src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -16,8 +16,29 @@
 public class HelpWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
-
+    public static final String HELP_MESSAGE = "List of commands: \n\n\n"
+            + "<<Viewing help>>\n"
+            + "Format: help\n\n\n"
+            + "<<Adding a person>>\n"
+            + "Format: add n/NAME p/PHONE_NUMBER e/EMAIL r/REMARK a/ADDRESS [t/TAG]\n\n\n"
+            + "<<Adding a remark>>\n"
+            + "Format: remark NRIC r/REMARK\n\n\n"
+            + "<<Listing all persons>>\n"
+            + "Format: list\n\n\n"
+            + "<<Adding an appointment>>\n"
+            + "Format: appointment NRIC app/APPOINTMENT\n\n\n"
```

Comment:
just to update Appointment is in format app/dd-MM-yyyy HH:MM, eg.
app/01-01-2025 12:34, probably good to specify